challenges/objects-arrays.js

```diff
@@ -88,32 +121,33 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
-const animalNames = [];
-console.log(animalNames);
+const animalNames = zooAnimals.forEach((animal) => console.log(animal.animal_name, animal.scientific_name));
 
 /* Request 2: .map()    
 
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
 
-const lowerCase = [];
+const lowerCase = zooAnimals.map((lowName) => lowName.animal_name.toLowerCase());
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
+const largerPopulation = zooAnimals.filter((animal) => animal.population < 5);
```

Comment:
Great use of the `filter` and knowing that the call back should evaluate out to a truth or false value, and that is what determines if the current item the `filter` is iterating over will make it into the array that the `filter` returns.